tests/fibshow_test.py

```diff
@@ -0,0 +1,92 @@
+import os
+
+import pytest
+
+from click.testing import CliRunner
+
+from utilities_common import multi_asic
+from utilities_common import constants
+from .utils import get_result_and_return_code
+import show.main as show
+
+from unittest.mock import patch
+
+from sonic_py_common import device_info
+
+show_ip_fib_v4 = """\
+  No.  Address             nexthop                                  ifname                                                       vrf
+-----  ------------------  ---------------------------------------  -----------------------------------------------------------  -----
+    1  192.168.104.0/25    10.0.0.57,10.0.0.59,10.0.0.61,10.0.0.63  PortChannel101,PortChannel102,PortChannel103,PortChannel104  False
+    2  192.168.104.128/25                                           PortChannel101,PortChannel102,PortChannel103,PortChannel104  False
+    3  192.168.112.0/25    10.0.0.57,10.0.0.59,10.0.0.61,10.0.0.63                                                               False
+    4  192.168.120.0/25    10.0.0.57,10.0.0.59,10.0.0.61,10.0.0.63  PortChannel101,PortChannel102,PortChannel103,PortChannel104  False
+    5  192.168.112.128/25  10.0.0.57,10.0.0.59,10.0.0.61,10.0.0.63  PortChannel101,PortChannel102,PortChannel103,PortChannel104  True
```

Comment:
@yutongzhang-microsoft , we may want to print the actual 'vrf' value instead of True/False. For emtpy Vrf, we can leave the field. Also I suggest to print Vrf as the second column and capitalize the headers

`No. Vrf Address Nexthop Ifname `

Reply:
I know, I will modify.

Reply:
done